editor/editor_properties.cpp

```diff
@@ -1320,7 +1320,31 @@ void EditorPropertyInteger::_value_changed(int64_t val) {
 }
 
 void EditorPropertyInteger::update_property() {
-	int64_t val = get_edited_property_value();
+	int64_t val = 0;
+	bool is_not_theme_property = true;
+	if (is_checkable() && !is_checked()) {
+		Control *control = Object::cast_to<Control>(get_edited_object());
+		if (control) {
+			String property_name = get_edited_property();
+			if (property_name.begins_with("theme_override_font_sizes/")) {
+				String name = property_name.substr(strlen("theme_override_font_sizes/"));
+				if (control->has_theme_font_size(name)) { // Exclude the dump font size cache.
+					val = control->get_theme_font_size(name);
+					is_not_theme_property = false;
+				}
```

Comment:
That's quite a bit repetitive code over 3 classes. Maybe it can be unified somehow.

Reply:
Added `Control::get_used_theme_item()`. It is now possible to query using the full theme property name.